app/views/thredded/posts_common/_mark_as_unread_link.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= link_to(t('thredded.topics.mark_as_unread'), post.mark_unread_path, method: :post, class:
```

Comment:
Please use `button_to` instead, so that it works without JavaScript:

```erb
<%= button_to t('thredded.topics.mark_as_unread'), post.mark_unread_path,
              form: {class: 'thredded--post--mark-as-unread'} %>
```